.github/workflows/release.yml

```
@@ -128,7 +128,10 @@ jobs:
 
   build_cuda_release:
     if: github.repository == 'ml-explore/mlx'
-    runs-on: ubuntu-22-large
+    strategy:
+      matrix:
+        arch: ['x86_64', 'aarch64']
+    runs-on: ${{ matrix.arch == 'x86_64' && 'ubuntu-22-large' || 'ubuntu-22-large-arm' }}
```

I would put the conditional logic in the matrix so it's easier to see:
```suggestion
        include:
          - host: ubuntu-22-large
            arch: x86_64
          - host: ubuntu-22-large-arm
            arch: aarch64
    runs-on: ${{ matrix.host }}
```